### PR TITLE
Add entry to reproduction log 

### PR DESCRIPTION
**Pull request by AliRanjbari**
Add an entry to reproduction log of `experiments-msmarco-passage`
Configuration:  
1. Operating System:  
   * Linux (Ubuntu 22.04)  
2. Development Environment:  
   * Python: Version 3.10.12  
   * Java: Version 11.0.20.1  
3. Hardware:  
   * RAM: 16 GB  
   * Processor: Intel Core i5 (11th generation)  
   * Storage: SSD drive  
   * GPU: NVIDIA GeForce RTX 3050  

**Additionally, I encountered an error related to the retrieval and evaluation phase in the _openai.py file. Specifically, I received an error stating that the "openai" module has no attribute error. To address this issue, I made a modification to the default value of the "errors" parameter in the `retry_with_delay` function from `openai.errors.RateLimitError` to `openai.RateLimitError`.**

### CODE REVIEW

**Comment by lintool** [comment]
Please fix the conflict and continue along the onboarding path until you reach the end of the exercises, and then we'll take a look all at once.

**Comment by lintool** [comment]
Please fix the conflict and continue along the onboarding path until you reach the end of the exercises, and then we'll take a look all at once.

**Comment by AliRanjbari** [comment]
I completed Experiment nfcorpus, both 1 and 2, and everything went well. I also added entries to the reproduction logs.

**Comment by lintool** [comment]
Please resolve conflict, ordering entries chronologically.

**Comment by AliRanjbari** [comment]
All conflicts have resolved.

File: docs/conceptual-framework.md [hunk 1]
```diff
@@ -294,4 +294,7 @@ Before you move on, however, add an entry in the "Reproduction Log" at the botto
...
 + Results reproduced by [@Melissa1412](https://github.com/Melissa1412) on 2023-11-05 (commit [`acd969f`](https://github.com/castorini/pyserini/commit/acd969f8f234126c272d70d55d047a3804b52ff8))
 + Results reproduced by [@salinaria](https://github.com/salinaria) on 2023-11-12 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))
-+ Results reproduced by [@oscarbelda86](https://github.com/oscarbelda86) on 2023-11-13 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))
\ No newline at end of file
++ Results reproduced by [@oscarbelda86](https://github.com/oscarbelda86) on 2023-11-13 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))
++ Results reproduced by [@oscarbelda86](https://github.com/oscarbelda86) on 2023-11-13 (commit [`086e16b`](https://github.com/castorini/pyserini/commit/086e16be28b7dc6022f8582dbd803824dc2c1ad2))
```
Comment (lintool) [inline docs/conceptual-framework.md:298]: why does this entry appear twice?

Comment (AliRanjbari) [reply docs/conceptual-framework.md]: It's probably my mistake to copy that entry another time. I removed the duplicate one.

**Review by lintool — APPROVED** [review]

**Pull request merged**